src/components/SearchSploit/SearchSploit.tsx

```diff
@@ -142,122 +122,124 @@ const SearchSploit = () => {
+                    <TextInput label={"Search Term"} {...form.getInputProps("searchTerm")} />
+                    <NativeSelect
+                        {...form.getInputProps("searchOption")}
+                        label={"Search Option"}
+                        data={searchOptions}
+                        placeholder={"Pick a Search option"}
+                    />
+                    <NativeSelect
+                        {...form.getInputProps("outputType")}
+                        label={"Output"}
+                        data={outputTypes}
+                        placeholder={"Select an Output"}
+                    />
+                    <NativeSelect
+                        {...form.getInputProps("nonSearch")}
+                        label={"Non-Searching"}
+                        data={nonSearchOptions}
+                        placeholder={"Select an option"}
+                    />
+                    <TextInput
+                        label={"EBD-ID"}
+                        description="Exploit Database ID: Required when using the 'Path' output or Non-Search options."
+                        {...form.getInputProps("ebdId")}
+                    />
+                    <Button type={"submit"}>Scan</Button>
+                    <Accordion>
+                        <Accordion.Item value="item-1">
+                            <Accordion.Control>Help:</Accordion.Control>
+                            <Accordion.Panel>
+                                <List>
+                                    <Text weight={700}>Search Options:</Text>
+                                    {searchOptions.map((option) => (
+                                        <List.Item key={option}>{option}</List.Item>
+                                    ))}
+                                </List>
+                                <List>
+                                    <Text weight={700}>Output:</Text>
+                                    {outputTypes.map((type) => (
+                                        <List.Item key={type}>{type}</List.Item>
+                                    ))}
+                                </List>
+                                <List>
+                                    <Text weight={700}>Non-Searching:</Text>
+                                    {nonSearchOptions.map((option) => (
+                                        <List.Item key={option}>{option}</List.Item>
+                                    ))}
+                                </List>
+                            </Accordion.Panel>
+                        </Accordion.Item>
+                    </Accordion>
+                    {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
+                    <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
+                </Stack>
+            </form>
+        </>
```

Comment:
Close RenderComponent here
```suggestion
        </RenderComponent>
```